api/client/swarm/join_token.go

```diff
@@ -58,17 +53,17 @@ func newJoinTokenCommand(dockerCli *client.DockerCli) *cobra.Command {
 			}
 
 			if quiet {
-				if args[0] == "worker" {
+				if worker {
 					fmt.Fprintln(dockerCli.Out(), swarm.JoinTokens.Worker)
-				} else if args[0] == "manager" {
+				} else {
 					fmt.Fprintln(dockerCli.Out(), swarm.JoinTokens.Manager)
 				}
 			} else {
 				info, err := client.Info(ctx)
 				if err != nil {
 					return err
 				}
-				return printJoinCommand(ctx, dockerCli, info.Swarm.NodeID, args[0] == "worker", args[0] == "manager")
+				return printJoinCommand(ctx, dockerCli, info.Swarm.NodeID, worker, manager)
```

Comment:
yes, I've been thinking about that as well; possibly showing instructions on how to get the manager join-token instead?